### DISCUSSION TITLE
improving code coverage

### DISCUSSION THREAD

**Discussion by reynoldsm88** [lum-ai/odinson#354] [General]
@marcovzla @danebell @kwalcock 

hey everyone, do you have any problems with me opening an issue to improve the drop in code coverage that resulted from my refactoring things for incremental indexing?

i can open the issue and have it assigned to myself, i'd like to see if i can get some of those numbers up

**Comment by kwalcock** [comment]
I'm not a gatekeeper or anything, but I certainly don't have any problems with it.  You might keep in mind that there is a PR pending with some tweaks to your code that could result in eventual merge conflicts.  Mostly I just wanted to respond before you thought that nobody was watching this.  Thanks for your help.

**Comment by marcovzla** [comment]
@reynoldsm88 Yes that would be great! please feel free to do it.
In case you can't assign it to yourself for some issue with permissions, let us know and we will do it for you.
